iothub/device/samples/PnpDeviceSamples/PnpDeviceSamples/SimpleThermostat/Models/SimpleThermostat.json

```diff
@@ -1,53 +0,0 @@
-﻿{
-  "@context": "dtmi:dtdl:context;2",
-  "@id": "dtmi:com:example:simplethermostat;1",
```

Comment:
Why is the name in consistent?
simplethermostat or thermostat

Reply:
nm

Reply:
this looks like an outdated commit, could you try refreshing

Reply:
It should be Thermostat